metadata.yaml

```diff
@@ -1,8 +1,8 @@
 service:
   full_name: "Amazon EventBridge"
   short_name: "EventBridge"
-  link: https://aws.amazon.com/eventbridge/"
-  documentation: https://docs.aws.amazon.com/eventbridge/latest/APIReference/Welcome.html"
+  link: "https://aws.amazon.com/eventbridge/"
+  documentation: "https://docs.aws.amazon.com/eventbridge/latest/APIReference/Welcome.html"
```

Comment:
Does this organically change for other partitions? e.g. https://docs.amazonaws.cn/en_us/eventbridge/latest/APIReference/Welcome.html

Reply:
Unlikely, this logic lives in `OLM` (Operator Hub) and we're just providing the bare minimal information here. Considering this non-blocking.